### PR TITLE
[fix] Added graphviz dependency (Resolves #637)

### LINKED ISSUE #637: graphviz needs to be in requirements.txt
`podio-vis` needs `graphviz`. If it is not there some tests will fail. We need to document this and also put `graphviz` into the `requirements.txt` file.

### PR DESCRIPTION
**Pull request by SaxenaAnushka102**
Added graphviz to requirements.txt

Resolves #637

BEGINRELEASENOTES
- Added missing dependency `graphviz` for `podio-vis` functionality in the documentation build process.

ENDRELEASENOTES

### CODE REVIEW

File: requirements.txt
Comment (tmadlener) [inline requirements.txt:5]:
```suggestion
graphviz
```

File: doc/requirements.txt
Comment (tmadlener) [inline doc/requirements.txt:23]: This actually needs to go [here](https://github.com/AIDASoft/podio/blob/master/doc/requirements.in) and then we use `pip-compile` to generate the actual `requirements.txt` from there.

**Review by tmadlener — COMMENTED** [review]
Thanks. A few minor comments.

File: doc/requirements.txt
Comment (SaxenaAnushka102) [reply doc/requirements.txt]: @tmadlener Thanks for explaining, I've updated `requirements.in` . I'm sorry for adding the dependency to an auto-generated file 😅

**Comment by tmadlener** [comment]
Sorry, this might have been a bit confusing if you are not completely familiar with podios structure. There are actually two `requirements.txt` files:
-  We have [this `requirements.txt`](https://github.com/AIDASoft/podio/blob/master/requirements.txt) at the top level. This is what is necessary to use podio and it's python bindings. 
- We have another one in [`doc/requirements.txt`](https://github.com/AIDASoft/podio/blob/master/doc/requirements.txt). This is the one that is generated from the `requirements.in` in the same folder. These are the dependencies to generate and build the documentation of podio.

Hence, what you want to change for fixing the issue is the first one and not the `requirements.in`.

File: doc/requirements.in
Comment (tmadlener) [inline doc/requirements.in:5]:
```suggestion
sphinx_copybutton
```
Simply to not touch this file in this PR.

File: requirements.txt
Comment (tmadlener) [inline requirements.txt:5]:
```suggestion
graphviz
```
Something in your editor settings seems to strip off the last newline in each file.

Comment (SaxenaAnushka102) [reply requirements.txt]: Yes, you're right. I'll check for the new lines from next time. Thanks!

File: requirements.txt
Comment (tmadlener) [inline requirements.txt:6]:
```suggestion
```

**Review by tmadlener — APPROVED** [review]
Thanks a lot for your work (and patience). Waiting for the CI to run to completion before merging.

**Pull request merged**
